internal/overlay/testdata/expectedWrapped.yaml

```diff
@@ -0,0 +1,21 @@
+openapi: "3.1.0"
+info:
+    title: "Test"
+    version: "0.1.0"
+    summary: "Test Summary"
+    description: "Some test description.\nAbout our test document."
+paths:
+    "/anything/selectGlobalServer":
+        x-my-ignore: {"servers": [{"url": "http://localhost:35123", "description": "The default server."}]}
```

Comment:
A peculiarity of this implementation that needs to be looked at at a later date. Only happens when outputting yaml from json input